server/app.js

```diff
@@ -27,9 +32,37 @@ const io = socketio(server, {
     origin: "*",
   },
 });
+io.use(ioCookieParser());
+io.use((socket, next) => {
```

Comment:
It would be a cleaner approach to write this socket auth middleware as a separate function, then pass it here instead.

Reply:
done ✔️